workflow-templates/psake-ci.yml

```diff
@@ -37,56 +26,51 @@ jobs:
     runs-on: [self-hosted, Windows]
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-      # Remove if you don't need to fetch all history for all branches and tags
-      with:
-        fetch-depth: 0
+      - name: Checkout
+        uses: actions/checkout@v2
+        # Remove if you don't need to fetch all history for all branches and tags
+        with:
+          fetch-depth: 0
 
-    # Remove this step if you are not using submodules
-    #
-    # Fetching submodules with the actions/checkout action fails because
-    # it generates an invalid Windows path when storing the public SSH key
-    - name: Update git submodules
-      run: git submodule update --init --recursive
+      # Remove this step if you are not using submodules
+      #
+      # Fetching submodules with the actions/checkout action fails because
+      # it generates an invalid Windows path when storing the public SSH key
+      - name: Update git submodules
+        run: git submodule update --init --recursive
 
-    - name: Run CI task using psake
-      run: |
-        $env:OCTOPUS_DEPLOY_APIKEY = "${{ secrets.OCTOPUS_DEPLOY_API_KEY }}"
-        psake ci
+      - name: Run CI task using psake
+        run: |
+          $env:OCTOPUS_DEPLOY_APIKEY = "${{ secrets.OCTOPUS_DEPLOY_API_KEY }}"
+          psake ci
 
-    ## Uncomment the next two steps if you want to create an artifact
-    ## If a git tag was not being pushed, then we create an artifact
-    ## that contains the built packages
-    ## ref: https://wiki.meniga.net/display/DEVOPS/Migrate+build+process+to+GitHub+Actions+using+Windows+self-hosted+runner#MigratebuildprocesstoGitHubActionsusingWindowsself-hostedrunner-Moreadvancedbuildsteps
-    # - name: Add git tag to environment
-    #   if: startsWith(github.ref, 'refs/tags') == 'false'
-    #   id: set-packageversion
-    #   run: |
-    #     psake Set-PackageVersionOutput
+      ## Uncomment the next two steps if you want to create an artifact
+      ## If a git tag was not being pushed, then we create an artifact
+      ## that contains the built packages
+      ## ref: https://wiki.meniga.net/display/DEVOPS/Migrate+build+process+to+GitHub+Actions+using+Windows+self-hosted+runner#MigratebuildprocesstoGitHubActionsusingWindowsself-hostedrunner-Moreadvancedbuildsteps
+      # - name: Add git tag to environment
```

Comment:
Should this be commented out? Package version is needed in the last step

Reply:
Please see my comment